plugins/mcreator-core/blockly/js/extensions_procedure_deps.js

```diff
@@ -74,17 +74,17 @@ function uniqueValueValidator(fieldName) {
 function firstFreeIndex(block, fieldName, index, valueProvider) {
     const values = [];
     for (let i = 0; block.getField(fieldName + i); i++) {
-        if (i === index)
+        if (index && i === index)

```

Comment:
Why was this reintroduced?

Reply:
In case someone needs this code for their own purposes and drops by, they would be able to copy this function in its initial state without having to search other commits.